src/Symfony/Component/TypeInfo/composer.json

```
@@ -36,11 +36,9 @@
     "conflict": {
         "phpstan/phpdoc-parser": "<1.0",
         "symfony/dependency-injection": "<6.4",
-        "symfony/doctrine-bridge": "7.1.*",
-        "symfony/framework-bundle": "7.1.*",
-        "symfony/property-info": "7.1.*",
-        "symfony/serializer": "7.1.*",
-        "symfony/validator": "7.1.*"
+        "symfony/property-info": "<7.1.9",
+        "symfony/serializer": "<7.1.9",
+        "symfony/validator": "<7.1.9"
```

```suggestion
        "symfony/property-info": ">=7.1,<7.1.9",
        "symfony/serializer": ">=7.1,<7.1.9",
        "symfony/validator": ">=7.1,<7.1.9"
```

but is this really needed?
I'd be happy to remove them, less things to maintain